table.cpp

```diff
@@ -95,12 +94,6 @@ TableCell::TableCell(float d, const std::string& c, int precision, const std::st
   academic_integrity = ai;
   event = e;
 
-  if (reason != "") {
-    hoverText = "class=\"hoverable-cell\" data-hover-text=\""+userName+" received a "+std::to_string(daysExtended)+" day extension due to "+reason+" on "+gID+"\" ";
-  } else {
-    hoverText = "class=\"hoverable-cell\" data-hover-text=\""+userName+" received a "+std::to_string(daysExtended)+" day extension without specified reason on "+gID+"\" ";
```

Comment:
looks like we lost the unspecified reason hover text in the refactor.  please re-add below

Reply:
Added in [77f1112](https://github.com/Submitty/RainbowGrades/pull/77/commits/77f1112c058f1ed921c2eadaf258bd66ae708776)